pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDReport.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cpd;
+
+import java.util.Iterator;
+import java.util.Map;
+
+public class CPDReport {
+    private final Iterator<Match> matches;
+    private final Map<String, Integer> numberOfTokensPerFile;
+
+    public CPDReport(final Iterator<Match> matches, final Map<String, Integer> numberOfTokensPerFile) {
```

Comment:
```suggestion
    CPDReport(final Iterator<Match> matches, final Map<String, Integer> numberOfTokensPerFile) {
```

Reply:
This change results in errors for me, since I can no longer access the `CPDReport` class from `CPDReportRenderer`. Do you think it's a problem if this class is publicly accessible?

Reply:
I just mean to hide the constructor, the class is still accessible, right?

Reply:
Ah, my bad, you're right!